v0.2.1/app-shell/lib/metadata-storage.js

```diff
@@ -105,7 +105,7 @@
         return;
       }
       let v = this._arc.context.findViewById(viewId);
-      if (v.toList) {
+      if (v && v.toList) {
```

Comment:
What was the call stack leading to this, or what steps did you manually take to create it?

Wondering whether we should log an error message with the view id as well as failing gracefully since this seems like something unexpected that a dev would potentially want to investigate and fix.

Reply:
Part of this is lost to time.. but, I wonder if this situation could be triggered due to the async nature of this part of the codebase? We may have a reference to a view (viewId), but the view might no longer be available?

If so, this seems safe enough. Otherwise we can leave it off (since I'm not seeing the bug right now).

Reply:
It seems worth merging but what do you think about adding a log message with the view id if !v?

Reply:
Yes! Sounds great.

Reply:
And it turned out to be interesting. I think these are more or less expected since I went and deleted a bunch of data from my profile in firebase.

MetadataStorage couldn't find view shared:PROFILE/[Post]/posts/ to clear
MetadataStorage couldn't find view shared:PROFILE/[Post]/ALL_posts/ to clear
MetadataStorage couldn't find view shared:PROFILE/Event/ to clear
MetadataStorage couldn't find view shared:PROFILE/[Event]/ to clear
MetadataStorage couldn't find view shared:PROFILE/Restaurant/ to clear
MetadataStorage couldn't find view shared:PROFILE/[Restaurant]/ to clear
MetadataStorage couldn't find view shared:AMKEY-KyUdop92YcQCGW9NnuU/[Post]/posts/ to clear
MetadataStorage couldn't find view shared:PROFILE/[Post]/ALL_posts/ to clear
MetadataStorage couldn't find view shared:AMKEY-KyYIi6jAZ3sX3Y62UTv/[Post]/posts/ to clear
MetadataStorage couldn't find view shared:PROFILE/[Post]/ALL_posts/ to clear